### PR TITLE
Nav view fix #45

### LINKED ISSUE #45: Build Nav view
Build navigation so that user can travel around the site

### PR DESCRIPTION
**Pull request by hsadler**
Add working nav view. There's still an issue with the home tab not being highlighted on load. Spent too long trying to figure this out without success. Nav's working fine though.

### CODE REVIEW

File: client/index.html [hunk 1]
```diff
@@ -7,16 +7,33 @@
     <script src="lib/angular/angular.js"></script>
     <script src="lib/angular-route/angular-route.js"></script>
     <script src="http://cdnjs.cloudflare.com/ajax/libs/angular-ui-router/0.2.10/angular-ui-router.js"></script>
+    <script src="lib/jquery/dist/jquery.min.js"></script>
```
Comment (cassiebeckley) [inline client/index.html]: What's the jquery include for?

**Comment by EricMFischer** [comment]
Looks good

File: client/index.html (hunk 1 above)
Comment (hsadler) [inline client/index.html]: I thought I needed it when I was trying to get the tab highlight working. I forgot to take it out after testing. Will change a push again.

**Comment by hsadler** [comment]
Did another push with merge conflicts resolved and the jquery script removed from index.html. Please review and I'll merge it. Thanks!

**Comment by cassiebeckley** [comment]
Looks good! :+1:

**Comment by EricMFischer** [comment]
Nice work!

**Pull request merged**